modules/simulate-network-policies-ng20.adoc

```diff
@@ -6,13 +6,17 @@
 = Simulating network policies from the network graph
 
 [role="_abstract"]
-Your current network policies might allow unneeded network communications.
-To simulate the effects of a new set of network policies, use the network policy simulator. For information about using the network policy simulator to generate policies, see "Generating network policies in the network graph."
+Your current network policies might allow unneeded network communications. You can also use the network policy generator to create network policies that restrict ingress traffic to the computed baselines for a set of deployments.
```

Comment:
I understand that in the context of this module's location, users are probably reading this after reading other sections about what the network policy generator can do, but this is technically a discrete piece of information and maybe users are jumping right to this section:

```suggestion
Your current network policies might allow unneeded network communications. You can use the network policy generator to create network policies that restrict ingress traffic to the computed baselines for a set of deployments.
```